test/e2e/subscription_e2e_test.go

```diff
@@ -1346,8 +1350,10 @@ var _ = Describe("Subscription", func() {
 		packageName3 := genName("cpackage")
 
 		namedStrategy := newNginxInstallStrategy((genName("dep")), permissions, nil)
-		depNamedStrategy := newNginxInstallStrategy((genName("dep")), permissions, nil)
-		depNamedStrategy2 := newNginxInstallStrategy((genName("dep")), permissions, nil)
+		depNamedStrategy := newNginxInstallStrategy((genName("dep")), permissions2, nil)
+		depNamedStrategy2 := newNginxInstallStrategy((genName("dep")), permissions3, nil)
+		depNamedStrategy3 := newNginxInstallStrategy((genName("dep")), permissions4, nil)
+		depNamedStrategy4 := newNginxInstallStrategy((genName("dep")), permissions5, nil)
```

Comment:
Is there an issue with shared ServiceAccounts? Why wasn't this an issue in 4.5 and 4.6?